lib/utils/copy-to-clipboard.ts

```diff
@@ -0,0 +1,8 @@
+export const handleCopyToClipboard = async (content: any) => {
+  const url = new URL(content, window.location.origin).toString();
+  try {
+    await navigator.clipboard.writeText(url);
+  } catch (error) {
+    console.log(error);
+  }
+};
```

Comment:
refactor (blocking): Narrow the type. The function is copying text to the clipboard, not for creating a URL and copying it. Provide a more specific error, e.g. "This browser does not support the clipboard."

nit: name the function `copyToCliboard`. No need to prefix it with `handle`.

```suggestion
export const copyToClipboard = async (content: string) => {
  try {
    await navigator.clipboard.writeText(content);
  } catch (error) {
    console.log(error);
  }
};
```

example usage

```javascript
const url = new URL(content, window.location.origin).toString();

copyToClipboard(url);
```